### ISSUE TITLE
Crashes right after start on XperiaX@SFOS3.2.1

### ISSUE THREAD

**Issue by Olf0** [black-sheep-dev/harbour-aenigma#19]
Installed via SailfishOS:Chum GUI client.
```
[nemo@sailfish ~]$ harbour-aenigma 
[D] unknown:0 - Using Wayland-EGL
[W] unknown:151 - file:///usr/share/harbour-aenigma/qml/harbour-aenigma.qml:151:30: Type GamePage unavailable 
         initialPage: Component { GamePage { } } 
                                  ^
[W] unknown:35 - file:///usr/share/harbour-aenigma/qml/pages/GamePage.qml:35:5: PageBusyIndicator is not a type 
         PageBusyIndicator { 
         ^
[nemo@sailfish ~]$ 
```
So i tried starting per `sailfish-qml`:
```
[nemo@sailfish ~]$ sailfish-qml harbour-aenigma 
[D] unknown:0 - Using Wayland-EGL
[W] unknown:6 - file:///usr/share/harbour-aenigma/qml/harbour-aenigma.qml:6:1: module "org.nubecula.aenigma" is not installed 
     import org.nubecula.aenigma 1.0 
     ^
[nemo@sailfish ~]$ 
```
<br />

Visually both ways of starting Aenigma provided the same result as starting per desktop icon: a completely white screen.

**Comment by black-sheep-dev** [comment]
It seems that PageBusyIndicator and DisplayBlanking are the show stoppers here.

Already replaced PageBusyIndicator with BusyIndicator, but need to find another solution for display blanking thing.
Will try out Monichs Sailfish OS tools.

**Comment by black-sheep-dev** [comment]
Released version 0.3.7 that should fix that

**Comment by Olf0** [comment]
Thank very much, but I am unable to find it in [SailfishOS:Chum testing](https://build.sailfishos.org/project/show/sailfishos:chum:testing) or at OpenRepos. The releases here at GitHub ar just source releases AFAICS.
I would be happy to try v0.3.7.
